lib/paper_trail/sinatra.rb

```diff
@@ -27,13 +31,31 @@ def user_for_paper_trail
       current_user
     end
 
+    def info_for_paper_trail
+      {}
+    end
+
+    def paper_trail_enabled_for_request
+      ::PaperTrail.enabled?
+    end
+
     private
 
     # Tells PaperTrail who is responsible for any changes that occur.
     def set_paper_trail_whodunnit
       @set_paper_trail_whodunnit_called = true
       ::PaperTrail.whodunnit = user_for_paper_trail if ::PaperTrail.enabled?
     end
+
+    def set_paper_trail_request_info
+      @set_paper_trail_request_info_called = true
```

Comment:
I don't think this instance variable is necessary. You were probably copying `@set_paper_trail_whodunnit_called` above. It, also, is not necessary. I think it was copied to this repository by mistake.